src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker/mediapicker.controller.js

```diff
@@ -4,23 +4,26 @@ angular.module('umbraco').controller("Umbraco.PropertyEditors.MediaPickerControl
     function ($rootScope, $scope, dialogService, entityResource, mediaResource, mediaHelper, $timeout, userService, $location, localizationService) {
 
         //check the pre-values for multi-picker
-        var multiPicker = $scope.model.config.multiPicker && $scope.model.config.multiPicker !== '0' ? true : false;
+        var multiPicker = $scope.model.config.maxNumber && parseInt($scope.model.config.maxNumber) > 1 ? true : false;

```

Comment:
Would probably be better to use this instead, so if the `maxNumber` config contains 0 (0 or no value could be considered as unlimeted) or any other value than `1` it is considered as a multipicker. Also note that I don't think the number prevalue editor prevent from entering negative number, even when number input has e.g. `min="0"` browsers still allow to enter `-1` or other negative numbers and the number prevalue editor also allow negative numbers, so I don't think it has server side validation except it is a valid integer.
```
var multiPicker = $scope.model.config.maxNumber && parseInt($scope.model.config.maxNumber) !== 1 ? true : false;
```